packages/workflow/src/features/registration/sms-notification-utils.ts

```diff
@@ -8,43 +8,40 @@
  *
  * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
  */
-import { APPLICATION_CONFIG_URL } from '@workflow/constants'
+import { COUNTRY_CONFIG_URL } from '@workflow/constants'
 import fetch from 'node-fetch'
 import { logger } from '@opencrvs/commons'
 
-export enum InformantNotificationName {
-  birthInProgressSMS = 'birthInProgressSMS',
-  birthDeclarationSMS = 'birthDeclarationSMS',
-  birthRegistrationSMS = 'birthRegistrationSMS',
-  birthRejectionSMS = 'birthRejectionSMS',
-  deathRegistrationSMS = 'deathRegistrationSMS',
-  deathInProgressSMS = 'deathInProgressSMS',
-  deathDeclarationSMS = 'deathDeclarationSMS',
-  deathRejectionSMS = 'deathRejectionSMS'
+type NotificationStatus = boolean | null
```

Comment:
Do we really to have `null` as a possible value? I know that you probably added it for __marriage__ declarations, but we could just use `false`. Or we could just not have `MARRIAGE` entry in the payload at all as well

Reply:
Yeah but I thought if someone missed something like `sent-for-review`, this null check could be useful. Originally I think it was meant to be that way because if one is not mentioned we take the default value as false.

Reply:
You were probably looking for something like this then
```
type EventNotification = {
  'sent-notification'?: boolean
  'sent-notification-for-review'?: boolean
  'sent-for-approval'?: boolean
  registered?: boolean
  'sent-for-updates'?: boolean
}
```
Which is kinda equivalent to what you had previously I guess